core/templates/components/forms/schema-based-editors/schema-based-list-editor.component.ts

```diff
@@ -243,16 +249,22 @@ implements ControlValueAccessor, Validator {
     if (this.validators) {
       for (var i = 0; i < this.validators.length; i++) {
         if (this.validators[i].id === 'has_length_at_most') {
-          this.maxListLength = this.validators[i].max_value;
+          let maxValue = this.validators[i].max_value;
+          this.maxListLength = maxValue !== undefined ? maxValue : null;
         } else if (this.validators[i].id === 'has_length_at_least') {
-          this.minListLength = this.validators[i].min_value;
+          let minValue = this.validators[i].min_value;
+          this.minListLength = minValue !== undefined ? minValue : null;
```

Comment:
When max/min length not selected it was pass as undefined(? property). But to make things more clear use null to indicate the same.